dataactbroker/handlers/fileHandler.py

```diff
@@ -738,6 +738,10 @@ def submit_detached_file(submission):
         if not submission.d2_submission:
             raise ResponseException("Submission is not a FABS submission", StatusCode.CLIENT_ERROR)
 
+        # Check to see if the submission is  being published
+        if submission.publish_status_id == PUBLISH_STATUS_DICT['publishing']:
+            raise ResponseException("Submission is being published", StatusCode.CLIENT_ERROR)
```

Comment:
Minor thing, "Submission is already being published." seems like more of an error message